crawl-ref/source/traps.cc

```diff
@@ -541,24 +552,26 @@ void trap_def::trigger(actor& triggerer)
     case TRAP_GOLUBRIA:
     {
         coord_def to = p;
-        if (_find_other_passage_side(to))
+        _passage_type search_result = _find_other_passage_side(to);
+        if (search_result == _PASSAGE_FREE)
         {
             if (you_trigger)
                 mpr("You enter the passage of Golubria.");
             else
                 simple_monster_message(*m, " enters the passage of Golubria.");
 
-            if (triggerer.move_to_pos(to))
-            {
-                if (you_trigger)
-                    place_cloud(CLOUD_TLOC_ENERGY, p, 1 + random2(3), &you);
-                else
-                    place_cloud(CLOUD_TLOC_ENERGY, p, 1 + random2(3), m);
-                trap_destroyed = true;
-                know_trap_destroyed = you_trigger;
-            }
-            else
-                mpr("But it is blocked!");
+            // Should always be true.
+            bool assert_check = triggerer.move_to_pos(to);
```

Comment:
Name should describe what the value means. I'd call it `moved`, since then `ASSERT(moved)` is quite clear.